examples/verify_addresses_from_csv/verify.py

```diff
@@ -46,55 +45,37 @@
     sys.exit(1)
 
 # output csv names
-success_filename = os.path.join(output_dir, 'success.csv')
-errors_filename = os.path.join(output_dir, 'errors.csv')
+verifications_filename = os.path.join(output_dir, 'verifications.csv')
```

Comment:
So does US verifications not have the same concept of "errors" as the old one did?